HW4_Toropov/protein_tools.py

```diff
@@ -0,0 +1,336 @@
+alphabet_protein = {
```

Comment:
Круто что вы вынесли эти переменные отдельно вначале вне функций. Несколько моментов:

1) Такие вещи в питоне называются "константами" и их принято называть целиком капсом
```suggestion
ALPHABET_PROTEIN = {
```
2) Тут название понятное, но кажется можно придумать что-то получше. Что-то типа `proteinogenic_aminoacids`.

3) На надо тут было разбирать каждую букву на отдальной строке... Понятно, что не все в одну строку, но по несколько АК на строке

Аналогично про все константы ниже